ui/components/app/modals/modal.js

```diff
@@ -159,6 +160,28 @@ const MODALS = {
     },
   },
 
+  IMPORT_TOKEN_EXIT_MODAL: {
+    contents: <ImportTokenExitModal />,
+    mobileModalStyle: {
+      width: '90%',
+      top: getEnvironmentType() === ENVIRONMENT_TYPE_POPUP ? '52vh' : '36.5vh',
+      borderRadius: '10px',
+    },
+    laptopModalStyle: {
+      borderRadius: '10px',
+      width:
+        getEnvironmentType() === ENVIRONMENT_TYPE_POPUP ? '357px' : '367px',
+      top: 'calc(33% + 45px)',
+      paddingLeft:
+        getEnvironmentType() === ENVIRONMENT_TYPE_POPUP ? '16px' : null,
+      paddingRight:
+        getEnvironmentType() === ENVIRONMENT_TYPE_POPUP ? '16px' : null,
+    },
+    contentStyle: {
+      borderRadius: '10px',
+    },
+  },
+
```

Comment:
I think we should use Modal component from DS for `ImportTokensExitModal`, if we use that, we won't to update this and it will handle all the css. Wdyt?

Reply:
updated 🙏